kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt

```diff
@@ -232,7 +236,7 @@ internal class CoroutinesTimeoutExtension internal constructor(
         }
         /* The extension was registered via an annotation; check that we succeeded in finding the annotation that led to
         the extension being registered and taking its parameters. */
-        if (testAnnotationOptional.isEmpty && classAnnotationOptional.isEmpty) {
+        if (!testAnnotationOptional.isPresent && !classAnnotationOptional.isPresent) {
```

Comment:
Could've it been caught with [`jdk-release`](https://jakewharton.com/kotlins-jdk-release-compatibility-flag/)?

If so, we can avoid setting up the integration tests for all the cases like this

Reply:
I've read the article, but I don't know how to get it to work. I add this to `kotlinx-coroutines-debug/build.gradle.kts`:
```kotlin
tasks.named<KotlinCompile>("compileKotlin") {
    compilerOptions {
        freeCompilerArgs.add("-Xjdk-release=1.8")
    }
}
```
(or `-Xjdk-release=8`), and I get errors:
<details>
  <summary>Symbol is declared in blah-blah, which the current module does not depend on.</summary>

```
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:1:7 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:12:60 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:18:9 Symbol is declared in module 'net.bytebuddy.agent', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:18:24 Symbol is declared in module 'net.bytebuddy.agent', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:18:32 Symbol is declared in module 'net.bytebuddy.agent', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:18:88 Symbol is declared in module 'net.bytebuddy.agent', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:19:18 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:19:24 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:20:19 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:20:25 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:22:9 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:23:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:24:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:24:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:25:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:26:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:26:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:26:35 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:26:58 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:30:18 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:30:24 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:31:19 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:31:25 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:32:9 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:33:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:34:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:34:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:35:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:36:14 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:36:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:36:35 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/Attach.kt:36:58 Symbol is declared in module 'net.bytebuddy', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:1:7 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:12:2 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:13:59 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:17:25 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:17:53 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:22:54 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:24:38 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:24:70 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:30:21 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:30:34 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:30:42 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:36:41 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:38:36 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:38:68 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:45:47 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:46:20 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:46:71 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:47:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:47:32 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:49:19 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:49:43 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:49:47 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:49:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:50:27 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:55:44 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:56:52 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:58:16 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:58:25 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:58:44 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:58:65 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:58:80 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:61:33 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:61:47 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:61:85 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:63:15 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:63:39 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:63:43 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:63:45 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:64:28 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutineInfo.kt:76:1 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:1:7 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:12:48 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:14:35 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:14:56 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:14:63 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:14:77 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:22:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:25:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:25:37 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:26:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:26:45 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:26:50 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:35:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:45:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:46:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:49:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:56:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:57:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:60:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:67:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:69:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:75:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:76:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:77:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:81:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:89:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:90:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:96:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:97:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:100:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:102:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:103:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:107:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:115:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:116:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:119:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:121:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:122:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:124:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:125:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:132:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:133:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:137:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:139:24 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:140:13 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:155:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:157:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:169:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:170:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:171:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:173:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:182:17 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt:183:9 Symbol is declared in module 'reactor.blockhound', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:1:7 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:49:2 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:60:17 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:60:33 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:61:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:63:13 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:63:29 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:75:17 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:75:33 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:76:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:78:13 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:78:29 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:92:17 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:92:33 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:93:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:95:13 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:95:29 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:101:45 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:101:61 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:108:9 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:108:25 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:115:9 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:115:25 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:121:52 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:134:33 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:134:48 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:134:64 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:140:37 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:141:21 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:141:27 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:141:44 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:141:52 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:146:30 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:146:40 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:146:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:146:61 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:146:67 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:147:13 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:147:21 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:147:37 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:153:34 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:153:55 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:153:69 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:153:76 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:153:82 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:154:18 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:154:24 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:154:41 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:154:49 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:161:9 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:161:25 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:161:46 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:161:50 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:36 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:50 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:57 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:63 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:70 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/DebugProbes.kt:181:86 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:1:7 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:10:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:11:47 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:11:66 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:12:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:13:49 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:13:68 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:14:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:15:56 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/NoOpProbes.kt:15:92 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:18:28 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:19:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:20:26 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:27:22 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:27:64 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:27:70 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:27:72 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:29:20 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:31:26 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:32:27 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:32:46 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:32:55 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:33:17 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:35:17 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:40:39 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:47:5 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:47:12 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:47:16 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:48:5 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:48:12 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:48:16 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:51:5 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:51:12 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:51:16 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:60:16 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:68:42 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:69:21 Symbol is declared in module 'kotlinx.coroutines.core', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:74:52 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/CoroutinesTimeoutImpl.kt:75:29 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:38:5 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:40:6 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:48:9 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:62:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:73:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:80:17 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:80:26 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:80:34 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:89:30 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:89:54 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeout.kt:89:68 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:9:32 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:10:34 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:13:5 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:17:61 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:18:19 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:18:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:18:65 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit4/CoroutinesTimeoutStatement.kt:20:31 Symbol is declared in module 'junit', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:50:2 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:51:2 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:52:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:53:2 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:53:44 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:53:63 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:54:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:54:20 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:54:40 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:55:2 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:55:9 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:55:26 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:55:33 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeout.kt:55:50 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:11:65 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:56:52 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:69:10 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:76:46 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:78:76 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:90:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:91:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:91:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:92:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:95:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:122:46 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:123:20 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:123:62 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:124:13 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:124:40 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:124:99 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:129:9 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:130:17 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:141:23 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:141:42 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:141:65 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:142:52 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:155:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:155:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:156:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:156:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:157:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:163:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:163:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:164:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:164:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:165:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:171:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:171:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:172:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:172:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:173:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:179:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:179:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:180:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:180:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:181:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:187:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:187:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:188:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:188:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:189:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:195:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:196:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:196:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:197:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:201:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:201:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:202:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:202:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:203:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:208:20 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:208:60 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:9 Symbol is declared in module 'org.junit.platform.commons', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:9 Return type mismatch: expected 'java.util.Optional<kotlinx.coroutines.debug.junit5.CoroutinesTimeout>', actual 'java.math.BigInteger'.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:27 Symbol is declared in module 'org.junit.platform.commons', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:73 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:79 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:209:79 Unresolved reference. None of the following candidates is applicable because of a receiver type mismatch:
@SinceKotlin(...) @InlineOnly() fun BigInteger.or(other: BigInteger): BigInteger
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:210:13 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:210:62 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:210:71 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:215:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:216:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:216:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:217:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:221:13 Symbol is declared in module 'org.junit.platform.commons', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:221:31 Symbol is declared in module 'org.junit.platform.commons', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:221:64 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:221:101 Symbol is declared in module 'kotlin.stdlib', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:222:56 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:222:66 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:225:40 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:225:77 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:229:23 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:231:70 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:231:81 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:235:36 Function invocation 'isEmpty()' expected.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:235:71 Function invocation 'isEmpty()' expected.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:236:19 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:239:36 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:240:57 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:241:67 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:241:78 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:244:59 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:245:58 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:246:67 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:246:78 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:250:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:256:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:257:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:257:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:258:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:262:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:262:54 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:263:28 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:263:56 Symbol is declared in an unnamed module which is not read by current module.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:264:27 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:268:21 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
e: file:///home/dkhalansky/IdeaProjects/kotlinx.coroutines/kotlinx-coroutines-debug/src/junit/junit5/CoroutinesTimeoutExtension.kt:274:73 Symbol is declared in module 'org.junit.jupiter.api', which the current module does not depend on.
```
</details>

Reply:
Looks like a related issue: https://youtrack.jetbrains.com/issue/KT-60868/JPMS-related-Symbol-is-declared-in-module-kotlin.stdlib-which-current-module-does-not-depend-on-when-using-Xjdk-release1.8